src/main/starlark/core/repositories/download.bzl

```diff
@@ -89,7 +89,7 @@ def kt_download_local_dev_dependencies():
     )
 
     rules_stardoc_repository(
-        name = "rules_android",
+        name = "build_bazel_rules_android",
```

Comment:
not sure of the context of this macro but it seems to basically be `http_archive` but adding some stardoc extras